src/suvela/assignment38/FindNuminFibonacci.java

```diff
@@ -0,0 +1,36 @@
+/*Assignment - 38 : 11th Dec'2020 
+Verify number is in Fibonacci series or not.
+
+*/
+package suvela.assignment38;
+
+public class FindNuminFibonacci {
+	void findNum(int num) {
+
+		int i = 0, j = 1, add = 0;
+
+		System.out.println("Fibonacci series: ");
+		System.out.print(i + "," + j);
+		while (add < num) {
+			add = i + j;
+			System.out.print("," + add);
+			i = j;
+			j = add;
+
+		}
+		System.out.println("-----------------");
+		if (add == num)
```

Comment:
this condition should be in while loop, think on this, let me know if you are not clear on this point.